### DISCUSSION TITLE
Microsite overview page documentation issue

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-Knowlg/Community#3] [Documentation Issues]
The [Overview page](https://knowlg.sunbird.org/learn/readme) spoke of diverse use cases, all subsequent pages went into learning use case alone. I suggest that overview page be corrected to talk about "content and knowledge" organization across LMS, Commerce, IoT domains rather than actual product such as footwear etc (instead "content about footwear" is what we are talking). That way, rest of the pages talking about "content organization" can somewhat makes sense for readers. 

@Krishnaj20 @vrayulu

**Comment by Krishnaj20** [comment]
The [Overview Page](https://knowlg.sunbird.org/learn/readme) is now corrected as per this suggestion.

**Reply by pramodkvarma** [reply]
Thanks. I am directly fixing few things myself, you can look at the history to see my updates. Also, @Krishnaj20 I am adding few comments on Gitbook when I see things to fix and tagging you. Hope you are receiving notifications (if not let me know). Please track those comments.
